params/params_joel.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 5
+    
+# e.g. "anas_krystallinus"    
+species.name <- "george "
```

Comment:
Got an extra line here...

```suggestion
species.name <- "george"
```